common/src/main/java/io/homeassistant/companion/android/common/assist/AssistViewModelBase.kt

```diff
@@ -114,21 +116,26 @@ abstract class AssistViewModelBase(
                     AssistPipelineEventType.STT_END -> {
                         stopRecording()
                         (it.data as? AssistPipelineSttEnd)?.sttOutput?.let { response ->
-                            onMessage(response["text"] as String, true, false)
+                            onMessage(response["text"] as String, true, false, false)
                         }
                     }
                     AssistPipelineEventType.INTENT_END -> {
                         val data = (it.data as? AssistPipelineIntentEnd)?.intentOutput ?: return@collect
                         conversationId = data.conversationId
+                        continueConversation.set(data.continueConversation)
                         data.response.speech.plain["speech"]?.let { response ->
-                            onMessage(response, false, false)
+                            onMessage(response, false, false, false)
                         }
                     }
                     AssistPipelineEventType.TTS_END -> {
                         if (!isVoice) return@collect
                         val audioPath = (it.data as? AssistPipelineTtsEnd)?.ttsOutput?.url
                         if (!audioPath.isNullOrBlank()) {
-                            playAudio(audioPath)
+                            playAudio(audioPath) {
+                                // We send the continueConversation flag here after getting it from INTENT_END so that
```

Comment:
Does `INTENT_END` refer to a pipeline stage here? Let's use the app's constants then to avoid confusion.